L/libpolymake_julia/common.jl

```diff
@@ -5,11 +5,12 @@ import Pkg: PackageSpec
 import Pkg.Types: VersionSpec
 
 name = "libpolymake_julia"
-version = v"0.3.0"
+const version = VersionNumber(0, 4, julia_version.minor)
```

Comment:
Note this quirky version -- we should have something newer than 0.3, so at least 0.4;  but at this point, we are decoupling from the upstream version anyway, so we might as well use 3.0 or 4.0 or whatever else strikes your fancy.

I would also in general like to float the idea of not bothering with on upstream `libpolymake-julia`, and instead use a `GitSource` below. At least I fail to see any benefit from having a version there? It just adds extra work?